vector/src/main/java/im/vector/riotx/features/home/HomeActivity.kt

```diff
@@ -144,6 +145,8 @@ class HomeActivity : VectorBaseActivity(), ToolbarConfigurable {
                 && activeSessionHolder.getSafeActiveSession()?.hasAlreadySynced() == true) {
             promptCompleteSecurityIfNeeded()
         }
+
+        shortcutsHandler.observeRoomsAndBuildShortcuts(context = this)
```

Comment:
Remove the context parameter and add a call to `disposeOnDestroy()` on the disposable